### PR TITLE
Remove unnecessary SDK exports

### PR DESCRIPTION
**Pull request by shrishrirang**
@danderson00

### CODE REVIEW

File: e2etest/tests/shared/mobileServiceTable.Functional.js [hunk 1]
```diff
@@ -26,15 +23,15 @@ function defineTableGenericFunctionalTestsNamespace() {
...
-            WindowsAzure.MobileServiceTable.SystemProperties.Version
+            WindowsAzure._MobileServiceTable.None,
+            WindowsAzure._MobileServiceTable.All,
+            WindowsAzure._MobileServiceTable.CreatedAt | WindowsAzure._MobileServiceTable.UpdatedAt | WindowsAzure._MobileServiceTable.Version,
+            WindowsAzure._MobileServiceTable.CreatedAt | WindowsAzure._MobileServiceTable.UpdatedAt,
+            WindowsAzure._MobileServiceTable.CreatedAt | WindowsAzure._MobileServiceTable.Version,
+            WindowsAzure._MobileServiceTable.CreatedAt,
+            WindowsAzure._MobileServiceTable.UpdatedAt | WindowsAzure._MobileServiceTable.Version,
+            WindowsAzure._MobileServiceTable.UpdatedAt,
+            WindowsAzure._MobileServiceTable.Version
```
Comment (danderson00) [inline e2etest/tests/shared/mobileServiceTable.Functional.js]: Why add the underscore? I guess it's kind of a convention in Javascript to indicate that the properties are "internal", but that's usually reserved for object instances. It's rare in pseudo-namespaces like this. I'd just leave it without the underscore. It's nasty.

**Comment by shrishrirang** [comment]
On closer look, I realized we need neither the underscored export nor the tests. These are stale tests from the time when we had to explicitly request the backend to send system properties. Updated the PR.

**Comment by danderson00** [comment]
Awesome love all that deleted code :D :shipit:

**Comment by jvano** [comment]
:shipit:

**Pull request merged**